operator/controllers/service_controller.go

```diff
@@ -68,50 +69,29 @@ func (r *ServiceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return reconcile.Result{}, fmt.Errorf("could not fetch Service: %+v", err)
 	}
 
-	if service.IsCompleted() {
-		return reconcile.Result{}, nil
-	}
-
 	logger.Info("Reconciling Service", "service", service.Name)
 
-	if service.IsServiceRunNotCreated() {
-		logger.Info("Service Run does not exist, creating")
-
-		serviceRunName, err := r.createServiceRun(ctx, service)
+	serviceRun := &v1alpha1.ServiceRun{}
+	err = r.Get(ctx, types.NamespacedName{Namespace: service.Namespace, Name: service.GetServiceRunName()}, serviceRun)
+	if errors.IsNotFound(err) {
+		err := r.createServiceRun(ctx, service)
```

Comment:
let's add a log message to indicate the successful creation of a service run

Reply:
I think this is worth an entry in the event stream of the service(workload) object ...

Reply:
there is an K8s event sent when starting the pre-deployment checks (creating KeptnEvent), should we create an event even if we create a new serviceRun? (instance)